### PR TITLE
Added util package

### PR DESCRIPTION
**Pull request by ppatierno**
This PR just adds an util package hosting just a common code about getting current time in milliseconds for now.

### CODE REVIEW

File: internal/util/util.go [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+// License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+//
+
+// Package util contains some utility functions
+package util
+
+import "time"
+
+// NowToMilliseconds returns the current time in milliseconds
+func NowToMilliseconds() int64 {
```
Comment (scholzj) [inline internal/util/util.go:12]: I wonder if it should be called `NowInMilliseconds()` => you are not converting some passed parameter to milliseconds but instead getting the current time in milliseconds.

**Review by scholzj — APPROVED** [review]
One nit, LGTM otherwise.

**Review by sknot-rh — APPROVED** [review]
Jakub's comment makes sense to me. Otherwise LGTM

File: internal/util/util.go (hunk 1 above)
Comment (ppatierno) [reply internal/util/util.go:12]: Agree, it makes more sense.

**Pull request merged**